### ISSUE TITLE
#3517.

### ISSUE THREAD

**Issue by berksiber** [ethereum/mist#3531]
- [x] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0
```

**Comment by berksiber** [comment]
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0

**Comment by ryanio** [comment]
Hey, we just released a new version (`0.11.0`) - please give it a try. If you have any new problems that you can reproduce, please create a new issue with the details on how we can help. Thanks!
